src/Statistics.ts

```diff
@@ -233,6 +234,23 @@ export const loadStatisticsCubeMultipliers = () => {
     }
 
     DOMCacheGetOrSet('sHeMT').textContent = `x${format(calculateHepteractMultiplier().mult, 3)}`;
+
+    const arr6 = calculateOcteractMultiplier().list;
+    const map6: Record<number, { acc: number, desc: string }> = {
+        1: {acc: 2, desc: 'Ascension Score Multiplier:'},
+        2: {acc: 2, desc: 'Global Cube Multiplier:'},
+        3: {acc: 2, desc: 'Season Pass 3:'},
+        4: {acc: 2, desc: 'Season Pass Lost:'},
+        5: {acc: 2, desc: 'Cookie Upgrade 20:'},
+        6: {acc: 2, desc: 'Divine Pack:'}
+    }
```

Comment:
change this to an array and remove the i + 1s

Reply:
I am unsure how to, I was just keeping the layout consistent with other cubes stat pages. I can look into it if necessary though.